grassvalley_scraper_working.py

```
@@ -0,0 +1,126 @@
+        'Sec-Fetch-Dest': 'document',
+        'Sec-Fetch-Mode': 'navigate',
+        'Sec-Fetch-Site': 'none',
+        'Sec-Ch-Ua': '"Not_A Brand";v="8", "Chromium";v="120", "Google Chrome";v="120"',
+        'Sec-Ch-Ua-Mobile': '?0',
+        'Sec-Ch-Ua-Platform': '"Linux"',
+        'Cache-Control': 'max-age=0'
+    }
+    
+    # Build URL for Grass Valley, CA search
+    base_url = "https://www.indeed.com/jobs"
+    params = {
```

Ruff (0.11.9)

Local variable `params` is assigned to but never used

Remove assignment to unused variable `params`

(F841)